client/blocks/reader-post-options-menu/blog-stickers.jsx

```diff
@@ -1,38 +1,29 @@
-import { map, includes } from 'lodash';
 import PropTypes from 'prop-types';
-import { Component } from 'react';
-import { connect } from 'react-redux';
-import QueryBlogStickers from 'calypso/components/data/query-blog-stickers';
-import getBlogStickers from 'calypso/state/selectors/get-blog-stickers';
+import { useBlogStickersQuery } from 'calypso/blocks/blog-stickers/use-blog-stickers-query';
 import ReaderPostOptionsMenuBlogStickerMenuItem from './blog-sticker-menu-item';
 
-class ReaderPostOptionsMenuBlogStickers extends Component {
-	static propTypes = {
-		blogId: PropTypes.number.isRequired,
-	};
+function ReaderPostOptionsMenuBlogStickers( { blogId } ) {
+	const blogStickersOffered = [ 'dont-recommend', 'broken-in-reader', 'a8c-test-blog' ];
```

Comment:
Could be declared outside of the component.

Reply:
Done via 95bbf4 👍🏻